condition/condition_plugins/include/condition_plugins/acceleration_condition.h

```diff
@@ -1,8 +1,6 @@
 #ifndef CONDITION_PLUGINS_ACCELERATION_CONDITION_H_INCLUDED
 #define CONDITION_PLUGINS_ACCELERATION_CONDITION_H_INCLUDED
 
-#include <pluginlib/class_list_macros.h>
```

Comment:
on another PR, I saw you changed it to `.hpp`. is that file not needed here at all? Then of course remove it

Reply:
Not in the header, I moved it to the cpp file.